import_upstream.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export PYTHONPATH=/home/rosbuild/reprepro_updater/src
```

Comment:
This repo originally was just a place for me to collect helper scripts. 

This master branch is only used for our legacy system. All scripts have the full paths hard coded and will never be released. There's the refactor branch which is where things have been reworked for potential release, and get their configuration from external parameterization.